.changeset/brave-chairs-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@codemod-utils/files": minor
+---
+
+Added a warning for findFiles()
```

Comment:
The change itself warrants a patch release. I went with a minor to account for a soft-breaking change in #229.